### DISCUSSION TITLE
[IntelliSense] Add a setting to force enable the IntelliSense extension

### DISCUSSION THREAD

**Discussion by AstragoDE** [tailwindlabs/tailwindcss#11617] [Ideas]
## Why should such a setting exist? 
I recently started using Tailwind CSS in Nuxt using the [Tailwind CSS Module](https://nuxt.com/modules/tailwindcss). Due to this, I do not necessarily need to have a `tailwind.config.js` file or similar file, but in order for the plugin to activate, I would need one.
In addition, I would see that such a setting would be useful, if I want automatic completion, when I just use the [Tailwind Play CDN](https://tailwindcss.com/docs/installation/play-cdn). 

## Why should it be an extension setting and not something else?
Settings for extensions can be set for different *levels*, meaning that VS Code settings can be changed in a per-workspace or even per-folder manner, and not only globally (the global force enable setting would be disabled by default).
![image](https://github.com/tailwindlabs/tailwindcss/assets/51675359/706c5ea5-98c6-4c35-b0b4-360dae27394d)
Changing a setting in the workspace section creates a `.vscode` folder in the project root / folder root with a file called `settings.json`, meaning that the workspace settings are saved in the project itself and therefore would be the same for other contributes.
The `settings.json` file **could** look like this to force enable IntelliSense:
```json
{
  "tailwindCSS.forceEnableIntelliSense": true
}
```
In addition to the above-mentioned advantage, I believe that implementing a force enable setting using the VS Code settings could be easier to implement and operate, and would conform to an already existing standard for changing settings for a specific workspace. 

---

Thank you very much for considering my idea for this feature.

**Comment by mybearworld** [comment]
As far as I can tell, the module uses TailwindCSS 2, you should probably not use it.

**Comment by AstragoDE** [comment]
Thank you for letting me know. I still think the above-mentioned feature should be implemented, since some people like to use the Play CDN.
